### PR TITLE
FAQ

### PR DESCRIPTION
**Pull request by amanuela97**
## FAQ Section

- [demo](https://wehelpdemo.netlify.app/)

### CODE REVIEW

File: components/block/faq.vue [hunk 1]
```diff
@@ -1,76 +1,117 @@
...
+              class="text-black hover:text-grey-500 underline"
+              href="https://drive.google.com/drive/folders/1MwIxcYWOrFmLnq4zMPejsGFF1SRHU7Fi)"
+              >Google Drive</a
+            >.</span
+          >
+        </FaqItem>
       </div>
     </div>
   </div>
 </template>
```
Comment (VOlekh) [inline components/block/faq.vue:117]: missing new line

File: components/faq/item.vue [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        <button
+          type="button"
+          class="text-left w-full flex justify-between items-start text-gray-400"
+          :aria-controls="`faq-${faqId}`"
+          :aria-expanded="collapse"
+          @click="switchCollapse"
+        >
+          <span
+            v-if="title"
+            class="font-nunito font-semibold text-[#2B2B2B] text-subtitle-2"
```
Comment (VOlekh) [inline components/faq/item.vue:15]: replace text-[#2B2B2B] with text-black-500

File: components/block/faq.vue [hunk 3]
```diff
@@ -1,76 +1,117 @@
 <template>
-  <div class="bg-neutral-300">
+  <div class="bg-gray-50">
     <div class="max-w-7xl mx-auto py-12 px-4 sm:py-16 sm:px-6 lg:px-8">
-      <div class="max-w-3xl mx-auto divide-y-2 divide-grey-200">
-        <h2
-          class="text-center text-3xl font-extrabold text-grey-800 sm:text-4xl"
-        >
-          Frequently asked questions
+      <div class="max-w-3xl mx-auto divide-y divide-[#C4C4C4]">
```
Comment (VOlekh) [inline components/block/faq.vue:4]: you have colors black-500, grey-300, neutral-500 take look on design in a black frame

File: components/faq/item.vue [hunk 4]
```diff
@@ -0,0 +1,71 @@
+<template>
+  <dl class="mt-6 space-y-6 divide-y divide-gray-200">
```
[the remark this answers is not in the document]
Comment (VOlekh) [reply components/faq/item.vue:2]: you only have  grey-300 in design

File: components/faq/item.vue (hunk 2 above)
Comment (VOlekh) [inline components/faq/item.vue:15]: color

File: components/faq/item.vue [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    <div class="pt-6">
+      <dt class="text-lg">
+        <!-- Expand/collapse question button -->
+        <button
+          type="button"
+          class="text-left w-full flex justify-between items-start text-gray-400"
```
Comment (VOlekh) [inline components/faq/item.vue:8]: color

**Review by VOlekh — APPROVED** [review]
Need to add empty lines and change colors

**Review by razbakov — CHANGES_REQUESTED** [review]
1. Please resolve solved conversations in PR
2. Please update demo link in PR description

**Review by razbakov — APPROVED** [review]

**Pull request merged**